### PR TITLE
Logging cleanup

### PR DESCRIPTION
**Pull request by smc395**
- Changed logging to warning for MentiiAuth
- Removed print statements from test
- Front end stuff was from view_class_details branch that Jon started on

### CODE REVIEW

File: Backend/mentii/class_ctrl.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import boto3
 import uuid
+import utils.MentiiLogging as MentiiLogging
```
Comment (asp78) [inline Backend/mentii/class_ctrl.py]: This is a duplicate import.

**Comment by asp78** [comment]
Ignore the second commit and revert...

File: Frontend/app/class/detail/detail.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <div class="row">
   <div class="col-xs-12">
-    <h2>Class Detail Page</h2>
+    <h3>[Class Name] <small>[Class Department]</small></h3>
+    <h4><small><strong>[Class Section]</strong></small></h4>
+    <p>[Class Description]</p>
```
Comment (RyanYoung25) [inline Frontend/app/class/detail/detail.html]: Why is this and the addition to class.service.ts on this pull request? What does it do?

Comment (smc395) [inline Frontend/app/class/detail/detail.html]: Removed from story

File: Frontend/app/class/detail/detail.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <div class="row">
```
Comment (jonathanmcdaniel) [inline Frontend/app/class/detail/detail.html]: This file isnt needed in this pull request

Comment (smc395) [inline Frontend/app/class/detail/detail.html]: Spacing issues only

**Pull request merged**